vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -345,42 +340,33 @@ and
 
         {new IProvideProjectSite with
             member iProvideProjectSite.GetProjectSite() =
-                let sources () = 
-                    let items,_,_ = projectInfoManager.GetProjectInfo(project.FilePath)
-                    items
-                let references () =
-                    let _,references,_ = projectInfoManager.GetProjectInfo(project.FilePath)
-                    references
-                let options () =
-                    let _,references,options = projectInfoManager.GetProjectInfo(project.FilePath)
-                    Array.concat [options; references |> Array.map(fun r -> "-r:" + r)]
-                let caption () = project.Name
-                let projFileName () = project.FilePath
-                let targetFrameworkMoniker = ""
-                let projectGuid () = project.Id.Id.ToString()
-                let creationTime = System.DateTime.Now
+                let fst  (a, _, _) = a
```

Comment:
Please call these ``p13`` and ``p33`` - that's what they are called in ``illib.fs`` of the F# compiler but I'm not sure you want to open that there in ``FSharp.Editor.dll``